### PR TITLE
feat : added like page and made it responsive

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
image

image

### CODE REVIEW

**Review by Mohasin-Haque — COMMENTED** [review]
Code looks good. Good folder structure. You can go ahead.

File: src/App.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ import { HomePage } from "./pages/homePage";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import {VideoListingPage} from "./pages/videoListingPage";
 import { VideoPlayerPage } from "./pages/videoPlayerPage";
```
Comment (t007rushi) [inline src/App.js:4]: barrel import it

File: src/Navbar/home/navbar.js [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
         )}{" "}
       </div>{" "}
-      <div className="ham-burger cursor" onClick={sidebarToggle}>
-        <i className="fa-solid fa-bars"> </i>{" "}
-      </div>{" "}
+      <img
+          src="https://i.ibb.co/9nWdYbV/Yash.jpg"
+          alt="profile-img"
+          className="video-listing_profile-img ml-1"
+        />{" "}
```
Comment (t007rushi) [inline src/Navbar/home/navbar.js:25]: there are many spaces which not needed like in routes can remove them

File: src/components/home/home.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import "./home.css";
 import { Link } from "react-router-dom";
-import { Sidebar } from "../../shared/sidebar/sidebar";
+// import { Sidebar } from "../../shared/sidebar/sidebar";
```
Comment (t007rushi) [inline src/components/home/home.js:3]: if not needed remove it

File: src/components/player/player.js [hunk 4]
```diff
@@ -22,13 +24,41 @@ const VideoPlayer = () => {
...
               className="react-player-frame"
               controls
+              
             />
           </div>
           <p className="about-videos">{description} </p>
-          <div className="flex-wrap-center pl-1 mr-1"> <caption className="mr-auto">{views}&nbsp;views</caption> <caption className="flex-wrap">< AiFillLike className="like-icon" size={25} />< MdWatchLater className="watchLater-icon" size={25} /> <MdPlaylistAdd className="playlist-icon" size={30} /></caption></div>
+          <div className="flex-wrap-center pl-1 mr-1">
+            {" "}
+            <div className="mr-auto">{views}&nbsp;views</div>
```
Comment (t007rushi) [inline src/components/player/player.js:36]: use Icons instead

**Review by t007rushi — APPROVED** [review]
if you want can create constants for type gives suggestions while code
else Looks Fine to merge with suggestions

**Pull request merged**